plugins/VersionUpgrade/VersionUpgrade46to47/__init__.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2020 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+
+from typing import Any, Dict, TYPE_CHECKING
+
+from . import VersionUpgrade46to47
+
+if TYPE_CHECKING:
+    from UM.Application import Application
+
+upgrade = VersionUpgrade46to47.VersionUpgrade46to47()
+
+def getMetaData() -> Dict[str, Any]:
+    return {
+        "version_upgrade": {
+            # From                           To                              Upgrade function
+            ("preferences", 6000012):        ("preferences", 6000013,        upgrade.upgradePreferences),
+            ("machine_stack", 4000012):      ("machine_stack", 4000013,      upgrade.upgradeStack),
+            ("extruder_train", 4000012):     ("extruder_train", 4000013,     upgrade.upgradeStack),
+            ("definition_changes", 4000012): ("definition_changes", 4000013, upgrade.upgradeInstanceContainer),
+            ("quality_changes", 4000012):    ("quality_changes", 4000013,    upgrade.upgradeInstanceContainer),
+            ("quality", 4000012):            ("quality", 4000013,            upgrade.upgradeInstanceContainer),
+            ("user", 4000012):               ("user", 4000013,               upgrade.upgradeInstanceContainer),
```

Comment:
Shouldn't there be an upgrader for Intent in this list as well?

Reply:
We've never had this, since the user can't create intent profiles unless by manually modifying his configuration folder.

Reply:
Ok, if it's on purpose, then I'm fine. We should _really_ keep an eye on this in case we ever make something like that though!